api/tests/serializers_tests.py

```diff
@@ -12,14 +13,22 @@
 
 
 class PremiumValidatorsTest(APITestCase):
+    def _get_token_for_user(self, user: User) -> Token:
+        if not hasattr(Token, "objects"):
```

Comment:
A possible docstring, if you need to describe the "why" of this code, and when it could be removed:

```suggestion
    def _get_token_for_user(self, user: User) -> Token:
        """
        Return DRF authentication Token for user, with type assertions.

        django-stubs 5.0.2 refactored how the Manager 'objects' attribute is added to
        Models, breaking djangorestframework-stubs 3.15.0. A fix [1] has been merged
        but not released, and there is a PR [2] reverting the objects refactor on
        django-stubs.

        mypy supports plugins to extend typing information [3], used by django-stubs
        to teach mypy about Django's model magic. The other major static type checker,
        pyright [4], does not and will not have this feature [5]. Adding 'hasattr'
        statements like this may be necessary to support pyright and Pylance [6],
        the best supported type checker for Visual Studio Code [7].

        [1]: https://github.com/typeddjango/djangorestframework-stubs/pull/644
        [2]: https://github.com/typeddjango/django-stubs/pull/2280
        [3]: https://mypy.readthedocs.io/en/stable/extending_mypy.html
        [4]: https://github.com/microsoft/pyright
        [5]: https://github.com/microsoft/pyright/blob/main/docs/mypy-comparison.md
        [6]: https://github.com/microsoft/pylance-release
        [7]: https://marketplace.visualstudio.com/items?itemName=ms-python.python
        """
        if not hasattr(Token, "objects"):
```